splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/RenameConstantOperation.java

```diff
@@ -284,7 +284,7 @@ public void executeConstantAction(Activation activation) throws StandardExceptio
 		td.setTableName(newTableName);
 		// add the table descriptor with new name
 		dd.addDescriptor(td, sd, DataDictionary.SYSTABLES_CATALOG_NUM,
-						 false, tc);
+						 false, tc, false);
```

Comment:
After talking to Mark, my above comment is not valid. The flag buildVersion2Desriptor should be  set to false, so that it can honor the tableVersion in td.